src/cluster/kind/mod.rs

```diff
@@ -90,14 +88,21 @@ impl KindCluster {
             .spawn()
             .context(IoSnafu {})?;
 
-        kind_cmd
-            .stdin
-            .as_ref()
-            .ok_or(ClusterError::StdinError)?
-            .write_all(config_string.as_bytes())
-            .context(IoSnafu {})?;
-
-        if let Err(err) = kind_cmd.wait() {
+        let mut stdin = kind_cmd.stdin.take().unwrap();
+        tokio::spawn(async move {
+            stdin.write_all(config_string.as_bytes()).await.unwrap();
+            drop(stdin);
+        });
```

Comment:
We shouldn't need to spawn a separate task for this, you're already waiting for the process to end below.